website/content/docs/platform/k8s/injector/examples.mdx

```diff
@@ -385,3 +385,36 @@ spec:
 
 [pkiCert]: https://github.com/hashicorp/consul-template/blob/main/docs/templating-language.md#pkicert
 [writeToFile]: https://github.com/hashicorp/consul-template/blob/main/docs/templating-language.md#writeToFile
+
+## Cross namespace secret sharing ((#cross-namespace))
+
+1. Configure Vault for secret sharing across namespaces as described in the
+   [How-to configure cross namespace access in Vault Enterprise][cross-namespace]
+   Knowledge base article.
+1. Use the following Pod annotations to authenticate to the Kubernetes method in
+   the `us-west-org` namespace, and render secrets from the `us-east-org`
+   namespace into the file `/vault/secrets/marketing`
+
+```yaml
+---
```

Comment:
Just confirming these dashes should be here since they aren't repeated at the end

Reply:
Yeah, they're not strictly necessary, but `---` is part of the yaml spec to signal the start of a yaml document.